lib/email-validation/strategy.rb

```diff
@@ -17,11 +17,10 @@ def verify_email(email)
         end
       end
 
-      validation_result ||= ValidationResult.new(true, true)
-
-      BlacklistedEmail.create(email: email, origin: validation_result.reason) unless validation_result.valid?
-
-      msg = validation_result.valid? ? '' : EmailValidation::incorrect_email_message(email)
+      unless validation_result.valid?
```

Comment:
Your link was about single line, mine was about block of code which is the case here. I know it's !! not ! but still that was my conclusion. But since we don't follow any style guide yet and I'm not a fan of 'unless' so let's do it if !validation_result.valid? then ;)